### ISSUE TITLE
[Feature Request] Advanced public status page

### ISSUE THREAD

**Issue by Tealk** [phpservermon/phpservermon#1101]
Hello,

would it be possible to upgrade the public page a little?
For example, that you can enter there planned maintenance or a history of the failures?

Or maybe there is a possibility to connect it with this project?
https://github.com/server-status-project/server-status

**Comment by Tealk** [comment]
An idea I would still have, that the public page is displayed as index and you can also log in there to then administer.

**Comment by CorneliousJD** [comment]
I am in no way affiliaited with this project, but I don't think the scope of this is to be a public status page. For that you could check out many other public status page projects. I've been using StatPing lately for that for anything that needs to be public facing.

That being said ,if they added a nice status page to PHPServerMon I'd be all for it :)

**Comment by TimZ99** [comment]
I agree with @CorneliousJD. Looking at the current backlog I think that this is too much work for now. Maybe this can be added in v4 👍🏼.

**Comment by Tealk** [comment]
is it possible to connect it to the following API?
https://github.com/server-status-project/server-status/wiki/API
